### DISCUSSION TITLE
Local-AI

### DISCUSSION THREAD

**Discussion by jboz** [quarkiverse/quarkus-langchain4j#676] [Q&A] [answered]
Is-it possible to use the [lanchain4j local-ai connector](https://docs.langchain4j.dev/integrations/language-models/local-ai) ?

**Comment by geoand** [comment, marked as answer]
You can add:

```xml
            <dependency>
                <groupId>dev.langchain4j</groupId>
                <artifactId>langchain4j-local-ai</artifactId>
            </dependency>
```

and manually instantiate `LocalAiChatModel` and the rest
